tests/testthat/_snaps/servers.md

```diff
@@ -15,15 +15,15 @@
     Output
       List of 3
        $ name       : chr "posit.cloud"
-       $ certificate: Amazon... (redacted)
        $ url        : chr "https://api.shinyapps.io/v1"
+       $ certificate: Amazon... (redacted)
```

Comment:
Is there any consequence to reordering this data frame?

Reply:
Shouldn't be — most uses will be via variable name, and this change makes the variable order consistent across all invocations.

Reply:
I like grouping the human-friendly fields together.